src/Database/PostUpdate.php

```diff
@@ -117,6 +118,9 @@ public static function update()
 		if (!self::update1484()) {
 			return false;
 		}
+		if (!self::update1506()) {
```

Comment:
If you add a method with a lower version number than the current highest, won’t it be skipped because the `post_update_version` will already be at 1507?

Reply:
No, this is not a problem. The post update version 1507 is introduced in this PR as well. And the previous post update version was 1484.

Reply:
Ok.